.github/workflows/deploy.yml

```
@@ -0,0 +1,75 @@
+      - name: connect ssh
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.DEV_HOST }}
+          username: ${{ secrets.DEV_USERNAME }}
+          password: ${{ secrets.DEV_PASSWORD }}
+          port: ${{ secrets.DEV_PORT }}
+          script: |
+            docker pull ${{ secrets.NCP_CONTAINER_REGISTRY }}/express2:${{ github.sha }}
+            docker stop $(docker ps -a -q)
+            docker rm $(docker ps -a -q)
+            docker run -d -p 8080:8080 --name app ${{ secrets.NCP_CONTAINER_REGISTRY }}/express2:${{ github.sha }}   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)